pkg/brokerinfo/rdsbrokerinfo_test.go

```diff
@@ -65,21 +65,22 @@ var _ = Describe("RDSBrokerInfo", func() {
 		It("returns error if it fails retrieving existing instances in AWS", func() {
 			fakeDBInstance.DescribeByTagError = fmt.Errorf("Error calling rds.DescribeByTag(...)")
 
-			_, err := brokerInfo.ListInstanceGUIDs()
+			_, err := brokerInfo.ListInstances()
 			Expect(err).To(HaveOccurred())
 		})
 		It("lists the instances for the right tag", func() {
-			_, err := brokerInfo.ListInstanceGUIDs()
+			_, err := brokerInfo.ListInstances()
 			Expect(err).NotTo(HaveOccurred())
 			Expect(fakeDBInstance.DescribeByTagKey).To(Equal("Broker Name"))
 			Expect(fakeDBInstance.DescribeByTagValue).To(Equal("broker_name"))
 		})
-		It("returns the list of instance GUIDs", func() {
-			instanceGUIDs, err := brokerInfo.ListInstanceGUIDs()
+		It("returns the list of instances", func() {
+			instanceGUIDs, err := brokerInfo.ListInstances()
```

Comment:
Minor: drop `GUID` from variable name as `ListInstances` now returns a collection of structs, rather than a `GUID` in a `string`